app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -26,20 +26,27 @@
     </div>
 
     <div id="mod-center-inbox" class="mod-center--nav--default crayons-tabs crayons-tabs--scrollable flex m:hidden">
-      <a
-        href="<%= "#{URL.url}/mod" %>"
-        class="crayons-tabs__item <%= "crayons-tabs__item--current" if @tag.blank? %>"
-        data-tag-name="All topics">
-        All topics
-      </a>
-      <% if @current_user_tags.any? %>
-        <% @current_user_tags.each do |tag| %>
-          <a href="<%= "#{URL.url}/mod/#{tag}" %>" class="crayons-tabs__item <%= "crayons-tabs__item--current" if tag == @tag&.name %>" data-tag-name="<%= tag %>">
-            #<%= tag %>
-            <span id="<%= tag %>-indicator" class="crayons-indicator"></span>
+      <ul class="crayons-tabs__list">
+        <li>
+          <a
+            href="<%= "/mod" %>"
```

Comment:
This has changed from `"#{URL.url}/mod"` to `"/mod"`, because otherwise Cypress tries to go to localhost instead of the port it's running on (where the session is), and the relative path is sufficient for the UI

Reply:
wouldn't that be slightly better: `<%= mod_path %>` ?

Reply:
100%! tbh I didn't know we could do that 😄  - thanks for pointing that out!